### ISSUE TITLE
to validating the DELETE POD failed

### ISSUE THREAD

**Issue by Sn0rt** [slok/kubewebhook#41]
hi, I want to delete pod safely.

so I set a validating  web hook, I have to try the kubewebhook.

however, the framework can't check the delete operation. basecase of the delete does not have a body only have a request. 

the code of deserializer doesn't take the request to the upper level.

```go
	// Get the object.
	_, _, err := w.deserializer.Decode(ar.Request.Object.Raw, nil, runtimeObj)
	if err != nil {
		err = fmt.Errorf("error deseralizing request raw object: %s", err)
		return w.toAdmissionErrorResponse(ar, err)
	}
```

**Comment by Sn0rt** [comment]
`admissionRequest := kubecontext.GetAdmissionRequest(ctx)`

**Comment by slok** [comment]
Reopening this because I'll close by linking the fix for the dynamic webhook issue (#63).

This is not needed anymore and the `DELETE` operations webhooks will come with the obj unmarshaled correctly, so there's no need to access the OldObject form the context review object.
